operating_unit/models/res_users.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent - Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp import api, fields, models
+
+
+class ResUsers(models.Model):
+
+    _inherit = 'res.users'
+
+    @api.model
+    def operating_unit_default_get(self, uid2):
+        if not uid2:
+            uid2 = self._uid
+        user = self.env['res.users'].browse(uid2)
+        return user.default_operating_unit_id and \
+            user.default_operating_unit_id.id
+
+    @api.model
+    def _operating_unit_default_get(self):
+        return self.operating_unit_default_get(self._uid)
+
+    @api.model
+    def _get_operating_units(self):
+        op_unit = self.operating_unit_default_get(self._uid)
+        if op_unit:
+            return [op_unit]
+        return False
+
+    operating_unit_ids = fields.Many2many(
+            'operating.unit', 'operating_unit_users_rel',
+            'user_id', 'poid', 'Operating Units',
+            default=_get_operating_units)
```

Comment:
@dreispt well, giving access to a user to several OU's is one of the foundations of the OU model. It is the application of your idea of having a user able to manage processes for multiple companies at the same time, without the need to change company (which is an absurd thing..).

If a user is allowed access to 3 OU's, then he can operate on them. The most important thing are the constraints to be checked. E.g. a user cannot create a sales order for OU1 and then an associated delivery order for OU2.